src/CrossValidation.h

```diff
@@ -70,10 +79,13 @@ class CrossValidation {
  protected:
   std::vector<AlgIn*> svmInputs_;
   std::vector< std::vector<double> > w_; // svm weights for each fold
+  std::vector<cpCnTriple> classWeightsPerFold_; // svm weights for each fold
```

Comment:
Does this replace the variable `w_` on the line above? Otherwise the comment is confusing.

Reply:
Comment was incorrect, updated it to "// cpos, cneg pairs to train for each nested CV fold"